index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Eze Promise</title>
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="wrapper">
+      <header>
+        <h1 class="logo">Welcome</h1>
+        <span class="menu">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </span>
+      </header>
+      <section class="headline">
+        <div class="intro">
+          <p>Hey there. I’m Promise <span class="title">I’m a software developer</span></p>
+          <p class="more-info"> I can help you build a product , feature or website Look through some of my work and experience!
```

Comment:
- [ ] Kindly align this paragraph to the left.
- [ ] Please change the` font-size` to the same `font-size `used in the Figma design.
- [ ] Please change the` line-height` to the same `line-height `used in the Figma design.

Reply:
In template 3, this paragraph was not aligned to the left.
The font size in the Figma design is 20 and that is what I used
The line-height in the Figma design is 28 and I used exactly that

Reply:
You are using flex to align the intro in the center of the element but if you check the original design the title is not in the middle. Please don't use align-item: center I think that will fix the issue.
 display: flex;
 flex-direction: column;
 padding: 0 10%;
 align-items: center;

Reply:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Reply:
Alright. Thank you